system/Email/Email.php

```diff
@@ -1080,10 +1087,8 @@ public function wordWrap($str, $charlim = null)
 
     /**
      * Build final headers
-     *
-     * @return void
      */
-    protected function buildHeaders()
+    protected function buildHeaders(): void
```

Comment:
Was this forced by one of our tools? Since this is considered a breaking change, I would rather avoid such a change.

Reply:
That's strange, I removed these return types and ran `composer cs` - no complaints.

Reply:
@ip-qi do not append the Email.php in your cs fix. just write `composer cs-fix`, run it and you're done

Reply:
Ok, seems to be an issue with the latest php-cs-fixer version. If so, run this instead for now:

utils/vendor/bin/php-cs-fixer fix -v system/Email/Email.php